src/Compilers/CSharp/Portable/Symbols/Synthesized/SynthesizedSimpleProgramEntryPointSymbol.cs

```diff
@@ -53,7 +55,7 @@ internal SynthesizedSimpleProgramEntryPointSymbol(SimpleProgramNamedTypeSymbol c
             bool isNullableAnalysisEnabled = IsNullableAnalysisEnabled(compilation, CompilationUnit);
             this.MakeFlags(
                 MethodKind.Ordinary,
-                DeclarationModifiers.Static | DeclarationModifiers.Private | (hasAwait ? DeclarationModifiers.Async : DeclarationModifiers.None),
+                DeclarationModifiers.Static | DeclarationModifiers.Public | (hasAwait ? DeclarationModifiers.Async : DeclarationModifiers.None),
```

Comment:
>Public [](http://example.com/codeflow?start=67&length=6)

Does the main method need to be public?

Reply:
We'd settled on public when we discussed a speakable method name. But we've reverted that decision since. I'll revert this to private, since it's unspeakable anyways.